packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -0,0 +1,366 @@
+
+          ['&:hover .' + globalClassnames.titleIsPlaceHolder]: rootHoverFocusActiveSelectorNeutralSecondaryMixin,
+          ['&:focus .' + globalClassnames.titleIsPlaceHolder]: rootHoverFocusActiveSelectorNeutralSecondaryMixin,
+          ['&:active .' + globalClassnames.titleIsPlaceHolder]: rootHoverFocusActiveSelectorNeutralSecondaryMixin,
+
+          ['&:hover .' + globalClassnames.titleHasError]: borderColorError,
+          ['&:active .' + globalClassnames.titleHasError]: borderColorError,
+          ['&:focus .' + globalClassnames.titleHasError]: borderColorError
+        }
+      },
+      className,
+      isOpen && 'is-open',
+      disabled && 'is-disabled',
+      required && 'is-required'
+    ],
+    title: [
+      globalClassnames.title,
+      normalize,
+      {
+        backgroundColor: theme.semanticColors.inputBackground,
+        borderWidth: 1,
+        borderStyle: 'solid',
+        borderColor: theme.semanticColors.inputBorder,
+        cursor: 'pointer',
+        display: 'block',
+        height: DROPDOWN_HEIGHT,
+        lineHeight: DROPDOWN_HEIGHT - 2,
+        padding: `0 ${DROPDOWN_HEIGHT}px 0 12px`,
+        position: 'relative',
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis'
+      },
+      isRenderingPlaceholder && [
+        globalClassnames.titleIsPlaceHolder,
+        {
+          color: theme.semanticColors.inputPlaceholderText
+        }
+      ],
+      hasError && [globalClassnames.titleHasError, borderColorError],
+      disabled && {
+        backgroundColor: theme.semanticColors.disabledBackground,
+        border: 'none',
+        color: theme.semanticColors.disabledText,
+        cursor: 'default',
+        selectors: {
+          [HighContrastSelector]: {
+            border: '1px solid GrayText',
+            color: 'GrayText'
+          }
+        }
+      }
+    ],
+    caretDownWrapper: [
+      {
+        position: 'absolute',
+        top: 1,
+        right: 12,
+        height: DROPDOWN_HEIGHT,
+        lineHeight: DROPDOWN_HEIGHT - 2 // height minus the border
+      }
+    ],
+    caretDown: [
+      globalClassnames.caretDown,
+      {
+        color: theme.palette.neutralSecondary,
+        fontSize: FontSizes.small,
+        pointerEvents: 'none'
+      },
+      disabled && {
+        color: theme.semanticColors.disabledText,
+        selectors: {
+          [HighContrastSelector]: {
+            color: 'GrayText'
+          }
+        }
+      }
+    ],
+    errorMessage: {
+      color: theme.semanticColors.errorText,
+      ...theme.fonts.small,
+      paddingTop: 5
+    },
+    callout: [
+      globalClassnames.callout,
+      {
+        boxShadow: '0 0 2px 0 rgba(0,0,0,0.2)',
+        border: `1px solid ${theme.palette.neutralLight}`
+      },
+      calloutClassName
+    ],
+    panel: [
+      globalClassnames.panel,
+      {
+        // TODO: use subcomponentstyles when panel is converted to use js styling.
```

Comment:
I don't think we have such a precedent.. and it fits into none of the existing 3 issue templates. How would you title the issue? `TODO: use subcomponentstyles in Dropdown when panel is converted to use js styling.`?

Reply:
It's a little awkward because it isn't really a bug - and we also have this problem (relying on selectors against global classnames) ALL over the place affecting a huge number of components.

Reply:
TODO is really weak :) We'll have TODO's all over the place in our code base unless we are really putting stuff on the books for this. It isn't a bug - it's a work item to be triaged in the backlog.